cns/restserver/ipam.go

```diff
@@ -35,6 +35,24 @@ func (service *HTTPRestService) requestIPConfigHandlerHelper(ipconfigsRequest cn
 		}, errors.New("failed to validate ip config request")
 	}
 
+	var MTpodIPInfo cns.PodIpInfo
+
+	// Request is for multitenant pod
+	if podInfo.IsMultitenant() {
+		// If pod is multitenant and we failed to grab its MTPNC IP config, return error immediately
+		podIPInfo, err := service.MultitenantMiddleware.GetSWIFTv2IPConfig(podInfo)
```

Comment:
it's probably not the responsibility of the multitenant "middleware" to do this and would be better organized as a different component

Reply:
my thinking is that this middleware will handle all related SWIFT v2 business, so it makes sense that it will handle returning the SWIFTv2 config as well..